### ISSUE TITLE
Script fails when fish is the default shell on remote machine

### ISSUE THREAD

**Issue by ghostbuster91** [gojuno/mainframer#43]

**Comment by artem-zinnatullin** [comment]
Any logs?

**Comment by ming13** [comment]
![](https://3.bp.blogspot.com/-XX-_2BguBCY/VtKBFupTW0I/AAAAAAAAEr0/cJ7PmeOq0MY/s1600/flash%2B23.gif)

**Comment by ming13** [comment]
I heard that `fish` [is not POSIX compliant](https://github.com/fish-shell/fish-shell/wiki/POSIX-compatibility-thoughts) (LOL). Probably shabang in the script is not enough and we must execute `bash mainframer.sh` explicitly.

**Comment by artem-zinnatullin** [comment]
I believe `rsync` will solve this as well, @ghostbuster91 please try 1.1.0 (will be released soon) and report back.

**Comment by ghostbuster91** [comment]
Thanks for quick replay.
Here is the error msg:
```
fish: set -xe && cd ~/el-peon-android/ && ./gradlew compileDebugUnitTestSources mockableAndroidJar
```
There are two problems with this command. 
1. Fish does not support `&&` for concatenation. Instead to be both fish and bash compatible you might want to use `;` character to separate commands.
2. The `set` command looks different in fish.

I think that the best solution would be to explicitly invoke these commands in bash shell, because supporting many shells might be cumbersome.

**Comment by artem-zinnatullin** [comment]
Yup, sounds reasonable

**Comment by artem-zinnatullin** [comment]
Closed by #44.
